docs/guides/bugzilla.rst

```diff
@@ -0,0 +1,192 @@
+            appliance,
+            virtualcenter_provider,
+            vm_compliance_policy_profile
+    ):
+        """
+        Tests to make sure that policy conditions with multiple or statements work properly
+
+        Bugzilla:
+            1711352
+            1717483
+
+        Polarion:
+            assignee: jdupuy
+            caseimportance: low
+            casecomponent: Control
+            initialEstimate: 1/12h
+        """
+        collection = appliance.provider_based_collection(virtualcenter_provider)
+        all_vms = collection.all()
+        all_vm_names = [vm.name for vm in all_vms]
+
+        # we need to select out cu-24x7
+        vm_name = virtualcenter_provider.data["cap_and_util"]["capandu_vm"]
+        # check that it exists on provider
+        if not virtualcenter_provider.mgmt.does_vm_exist(vm_name):
+            pytest.skip("No capandu_vm available on virtualcenter_provider of name {}".format(vm_name))
+
+        vms = [all_vms.pop(all_vm_names.index(vm_name))]
+
+        # do not run the policy simulation against more that 4 VMs
+        try:
+            vms.extend(all_vms[0:min(random.randint(1, len(all_vms)), 4)])
+        except ValueError:
+            pytest.skip("No other vms exist on provider to run policy simulation against.")
+
+        filtered_collection = collection.filter({"names": [vm.name for vm in vms]})
+        # Now perform the policy simulation
+        view = navigate_to(filtered_collection, "PolicySimulation")
+        # Select the correct policy profile
+        view.fill({"form": {"policy_profile": "{}".format(vm_compliance_policy_profile.description)}})
+
+        # Now check each quadicon and ensure that only cu-24x7 is compliant
+        for entity in view.form.entities.get_all():
+            state = entity.data["quad"]["bottomRight"]["tooltip"]
+            if entity.name == vm_name:
+                assert state == "Policy simulation successful."
+            else:
+                assert state == "Policy simulation failed with: false"
+
+This a nice test case because it combines several of the things above. It is `blocked`
+by the BZ 1717483, but it is providing automated test coverage for BZ 1711352. Note that it isn't
+providing coverage for BZ 1717483, so that BZ must be blocking a setup or teardown step of
+the test case. It also makes use of the ``Bugzilla`` docblock, discussed below.
+You can then run the following command to set qe_test_coverage to ``+`` for BZ 1711352.
+
+``miq bz coverage --set cfme/tests/control/test_bugs.py``
+
+A dry run of this command (i.e. without ``--set``) produces the following output:
+
+.. code-block:: console
+
+    The following BZs should have qe_test_coverage set to '+':
+        id: 1155284, qe_test_coverage: ?
+        id: 1243357, qe_test_coverage: ?
+        id: 1711352, qe_test_coverage: -
+
+Bugzilla docblock
+^^^^^^^^^^^^^^^^^
+The ``Bugzilla`` docblock is for listing any BZ's that are tangentially related to a test case.
```

Comment:
Can you please elaborate why `tangentially` here?

Reply:
by tangentially, I mean touching, like a test case may hit a BZ in a different FA, but the test case is not meant to test that BZ.